src/owasp/owasp-suppression.xml

```diff
@@ -76,5 +76,19 @@
         <gav regex="true">^org\.slf4j:log4j-over-slf4j:.*$</gav>
         <cve>CVE-2018-8088</cve>
     </suppress>
-
+    <suppress>
```

Comment:
This is a false positive as noted in https://github.com/dropwizard/dropwizard/issues/3185.